src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -466,7 +476,7 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
                         {
                             // we don't show the result tabs if we don't have valid query
                             (this.showTabs && !this.resultsViewPageStore.genesInvalid && !this.resultsViewPageStore.isQueryInvalid) && (
-                                <MSKTabs key={this.resultsViewPageStore.rvQuery.hash} activeTabId={this.currentTab(this.resultsViewPageStore.tabId)} unmountOnHide={false}
+                                <MSKTabs activeTabId={this.currentTab(this.resultsViewPageStore.tabId)} unmountOnHide={false}
```

Comment:
@adam I took the key off of this.  We need to discuss why we had key in the first place.  Obviously to force re-mounting.  But why?  There was a reason!

Reply:
man I wish we'd left a comment. clearly there was a reason because we went to the trouble of creating that hash.